service/build.gradle

```diff
@@ -168,13 +168,16 @@ def versions = [
 dependencies {
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.8'
   compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.8'
-  
+
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
 
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: versions.springBoot
 
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.1.2.RELEASE'
 
+//   The below fixes https://nvd.nist.gov/vuln/detail/CVE-2019-12384 while waiting for spring to pull new version
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.9.3'
```

Comment:
There are few suppressions in fpl-ccd-configuration/config/owasp/suppressions.xml. `can you double check whether we still need these. I suspect couple if these were fixed but library bump.

Reply:
Done, pushed new commit.